src/python/geoclaw/surge/storm.py

```diff
@@ -649,60 +660,81 @@ def write_geoclaw(self, path, verbose=False, max_wind_radius_fill=None,
          - *max_wind_radius_fill* (func) Function that can be used to fill in
            missing data for `max_wind_radius` values.  This defaults to simply 
            setting the value to -1.  The function signature should be
-           `max_wind_radius(t, storm)` whre t is the time of the forecast and
-           `storm` is the storm object.
+           `max_wind_radius(t, storm)` where t is the time of the forecast and
+           `storm` is the storm object.  Note that if this or `storm_radius` 
+           field remains -1 that this data line will be assumed to be redundant 
+           and not be written out.
          - *storm_radius_fill* (func) Function that can be used to fill in
            missing data for `storm_radius` values.  This defaults to simply 
            setting the value to -1.  The function signature should be
-           `storm_radius(t, storm)` whre t is the time of the forecast and
-           `storm` is the storm object.
+           `storm_radius(t, storm)` where t is the time of the forecast and
+           `storm` is the storm object.  Note that if this or `max_wind_radius` 
+           field remains -1 that this data line will be assumed to be redundant 
+           and not be written 
         """
 
         if max_wind_radius_fill is None:
             max_wind_radius_fill = lambda t, storm: -1
         if storm_radius_fill is None:
             storm_radius_fill = lambda t, storm: -1
 
-        # Remove duplicate times
-        num_casts = len(self.t)
-        for n in range(1, len(self.t)):
-            if self.t[n] == self.t[n - 1]:
-                num_casts -= 1
+        # Create list for output
+        # Leave this first line blank as we need to count the actual valid lines
+        # that will be left in the file below
+        num_casts = 0
+        data_string = [""]
+        if self.time_offset is None:
+            # Use the first time in sequence if not provided
+            self.time_offset = self.t[0]
+        import pdb; pdb.set_trace()
```

Comment:
This pdb statement is causing the travis failure.

Reply:
Thanks for catching that.